### PR TITLE
[Feat] [snt9|2_5] Add a case: GoogleNet-Based Flower recognition

### PR DESCRIPTION
**Pull request by ilovexsir**
## Test environment
Hardware: 1*ascend-snt9|ARM: 24核 96GB
Image: python3.9-ms2.5.0-cann8.0.0.beta1
## Test Report

Accuracy: 78%
Inference Time: 312 ms per image
Training Time: 28 minutes
Total Script Runtime: 34 minutes

## Related Issues

Fixes #4

### CODE REVIEW

File: dev/ascend/snt9/2_5/GoogleNet实现花卉识别.ipynb [hunk 1]
```diff
@@ -0,0 +1,794 @@
...
+    "ascend_target = (ms.get_context(\"device_target\") == \"Ascend\")\n",
+    "if ascend_target:\n",
+    "    model = train.Model(network, loss_fn=network_loss, optimizer=network_opt, metrics={\"acc\"}, amp_level=\"O2\")\n",
+    "else:\n",
+    "    model = train.Model(network, loss_fn=network_loss, optimizer=network_opt, metrics={\"acc\"}, amp_level=\"O0\")\n",
+    "\n",
+    "# train model\n",
+    "model.train(epoch_size,\n",
+    "            dataset_train,\n",
+    "            callbacks=[ckpt_callback, LossMonitor(125), TimeMonitor(125)],\n",
```
Comment (hongziqi) [inline dev/ascend/snt9/2_5/GoogleNet实现花卉识别.ipynb:609]:
```suggestion
    "            callbacks=[ckpt_callback, LossMonitor(500), TimeMonitor(500)],\n",
```
print loss pre 500 steps maybe better

File: dev/ascend/snt9/2_5/GoogleNet实现花卉识别.ipynb [hunk 2]
```diff
@@ -0,0 +1,794 @@
...
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   },
+   "source": [
+    "从结果可以看出，由于我们加载了预训练模型参数，模型的Top_1_Accuracy和Top_5_Accuracy达到了很高的水平，实际项目中也可以以此准确率为标准。如果未使用预训练模型参数，则需要更多的epoch来训练。"
```
Comment (hongziqi) [inline dev/ascend/snt9/2_5/GoogleNet实现花卉识别.ipynb:697]:
```suggestion
```

Comment (ilovexsir) [reply dev/ascend/snt9/2_5/GoogleNet实现花卉识别.ipynb]: I update it, please review it.

**Comment by hongziqi** [comment]
> ## PR Type
> > Please uncomment only one of the following lines, remove the leading spaces, and leave the other options commented out:
> > /kind ascend-case
> 
> ## Description of Changes
> ## Test environment
> Hardware: 1*ascend-snt9|ARM: 24核 96GB Image: python3.9-ms2.5.0-cann8.0.0.beta1
> 
> ## Test Report
> Accuracy: 78% Inference Time: 312 ms per image Training Time: 28 minutes Total Script Runtime: 2h 35 minutes
> 
> ## Related Issues
> Fixes #4
> 
> ## Additional Notes

Please update the `Total Script Runtime`

**Review by hongziqi — APPROVED** [review]
